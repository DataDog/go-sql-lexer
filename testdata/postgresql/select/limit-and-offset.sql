SELECT * FROM users ORDER BY created_at DESC LIMIT 10 OFFSET 20;
