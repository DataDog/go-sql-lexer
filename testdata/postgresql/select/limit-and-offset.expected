[
    {
        "expected": "SELECT * FROM users ORDER BY created_at DESC LIMIT ? OFFSET ?",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "SELECT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]